klimenko/Lesson14/src/main/java/com/klimenko/Lesson14/config/HibernateConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.klimenko.Lesson14.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+
+@Configuration
+@EnableTransactionManagement
+@PropertySource(value = {
+        "classpath:application.properties"
+})
+public class HibernateConfig {
+    @Autowired
+    private Environment env;
+
+    @Bean
+    public LocalSessionFactoryBean sessionFactory() {
+        LocalSessionFactoryBean factoryBean = new LocalSessionFactoryBean();
+
+        Properties props = new Properties();
+
+        props.put("hibernate.connection.driver_class", env.getProperty("jdbc.driver"));
```

Comment:
I believe with spring boot you can use this properties in application.properties